tests/kv_test/README.md

```diff
@@ -0,0 +1,4 @@
+To train and test RMI locally:
+```
+make result_all ROOT=./local-test RELOAD=./reload_local.sh STORAGE=storage-most OUT=out-most
```

Comment:
Let's use a hypothetical directory (so readers can substitute the paths with theirs). For example, say `ROOT=/path/to/test/dir` instead.